util/filter_bench.cc

```diff
@@ -606,7 +606,8 @@ double FilterBench::RandomQueryTest(uint32_t inside_threshold, bool dry_run,
   }
 
   if (!dry_run) {
-    fp_rate_report_ = std::ostringstream();
+    fp_rate_report_.clear();
+    fp_rate_report_.str("");
```

Comment:
ditto.

Reply:
Reson as above, and here is the conversation with @pdillinger https://github.com/facebook/rocksdb/pull/5968.